lib/run/index.js

```diff
@@ -103,23 +104,19 @@ module.exports = function (options, callback) {
             },
             // todo: add support for more types of timeouts, currently only request is supported
             timeout: options.timeoutRequest ? { request: options.timeoutRequest } : undefined,
+            fileResolver: fs,
             requester: {
-                fileResolver: fs,
                 cookieJar: request.jar(),
                 followRedirects: _.has(options, 'ignoreRedirects') ? !options.ignoreRedirects : undefined,
-                strictSSL: _.has(options, 'insecure') ? !options.insecure : undefined,
-                certificateManager: options.sslClientCert ? {
-                    getCertificateContents: function (hostname, callback) {
-                        return callback(null, {
-                            key: options.sslClientKey ? fs.readFileSync(options.sslClientKey) : '',
-                            pem: fs.readFileSync(options.sslClientCert),
-                            passphrase: options.sslClientPassphrase,
-                            keyPath: options.sslClientKey,
-                            pemPath: options.sslClientCert
-                        });
-                    }
-                } : undefined
-            }
+                strictSSL: _.has(options, 'insecure') ? !options.insecure : undefined
+            },
+            certificates: options.sslClientCert && new sdk.CertificateList({}, [{
+                name: 'client-cert',
+                matches: ['<all_urls>'],
```

Comment:
Pretty unhappy about having to do this, @kamalaknn 

I think this should be exposed as a constant from the SDK